docs/tutorial.rst

```diff
@@ -52,6 +52,10 @@ means we will minimize the sum of
 
 Building the model
 ------------------
+Blocks uses "bricks" to build models. Bricks are **parametrized Theano 
+operations**. You can read more about it in the 
+:ref:`building with bricks <bricks>` tutorial.
```

Comment:
Yes, it points to the anchor in https://github.com/bartvm/blocks/pull/473/files#diff-091662f3fd06a0802f6e2d6e9a3edfe0R11
It would be probably better to put the anchor at line 8 though, so that you see the subsection title. I will modify that in a minute.